src/components/Match/matchColumns.jsx

```diff
@@ -39,6 +39,7 @@ export default (strings) => {
     return (<TableHeroImage
       title={row.name || row.personaname || strings.general_anonymous}
       registered={row.last_login}
+      contributor={row.isContributor}
```

Comment:
update to snake_case

Reply:
eh, let's just keep it consistent with the other is_contributor.  I think that's better to be consistent rather than with other properties.